Source/WebCore/cssjit/SelectorCompiler.cpp

```diff
@@ -1225,6 +1225,7 @@ static inline FunctionType addPseudoClassType(const CSSSelector& selector, Selec
     case CSSSelector::PseudoClass::WebKitDrag:
     case CSSSelector::PseudoClass::Has:
     case CSSSelector::PseudoClass::State:
+    case CSSSelector::PseudoClass::ActiveViewTransitionType:
```

Comment:
Can you file a bug about adding CSSJIT support?

Reply:
https://bugs.webkit.org/show_bug.cgi?id=278189
I'll add it to the code too